pkgs/applications/science/electronics/verilator/default.nix

```diff
@@ -26,6 +26,13 @@ stdenv.mkDerivation rec {
       src/vlcovgen
   '';
 
+  # Tools that are built during build phase, but need to be manually linked or
+  # copied to the correct location.
+  postInstall = ''
+    ln -s $out/share/verilator/include $out/include
+    ln -s $out/share/verilator/bin/verilator_includer $out/bin/verilator_includer
```

Comment:
why not move them to the right location?

Reply:
I wasn't sure that the locations they end up were actually right, so I symlinked them, just to be safe.